hokusai/development.yml

```diff
@@ -4,8 +4,8 @@ services:
 {% include 'templates/docker-compose-service-dev.yml.j2' %}
     command: ["yarn", "start"]
     env_file:
-      - ../.env.development
       - ../.env.shared
+      - ../.env.development
```

Comment:
By my understanding we want to make this override `.env*`, correct? But not `.env`

Reply:
My original thinking is it would be nice if it also overrides `.env` so we always know whatever is in that file will be applied to local development. But I don't think in the repos I've seen so far it would matter too much, as `.env` is pretty empty.

Reply:
I vote to keep `.env` as is because it's for personal overrides and that applies to any environment. As you say, it wouldn't matter much since those files tend to be empty but if someone needs to override a development var, they can use `.env`.

Reply:
fair point. Agreed.